src/popup/components/BusinessCard/BusinessCard.js

```diff
@@ -16,7 +18,12 @@ const BusinessCard = props => {
       <Card className="mx-auto">
         <Card.Body>
           <Card.Title>{props.title}</Card.Title>
-          <Card.Text>{props.description}</Card.Text>
+          <Card.Text
+            onMouseEnter={() => setIsShown(true)}
+            onMouseLeave={() => setIsShown(false)}
```

Comment:
I think the event listener for mouseLeave will beed to be on the parent element for this component. Otherwise the description will minimize when users hover their mouse over the website for the button

Reply:
I agree--just added those changes. Thanks!

Reply:
Nice! Seems good.